src/csv_to_engagement_db/configuration.py

```diff
@@ -1,12 +1,88 @@
+from datetime import datetime
+
+import pytz
+
+
+_MIN_DATE_UTC = pytz.timezone("utc").localize(datetime.min)
+_MAX_DATE_UTC = pytz.timezone("utc").localize(datetime.max)
+
+
+class CSVDatasetConfiguration:
+    def __init__(self, engagement_db_dataset, start_date=_MIN_DATE_UTC, end_date=_MAX_DATE_UTC):
+        """
+        Configuration for an engagement db dataset to sync csv messages to.
+
+        To restrict messages to this dataset by message timestamps, optionally set the `start_date`/`end_date`
+        properties.
+
+        :param engagement_db_dataset: Name of the dataset to use in the engagement database.
+        :type engagement_db_dataset: str
+        :param start_date: Start date for this dataset configuration.
+        :type start_date: datetime.datetime
+        :param end_date: End date for this dataset configuration.
+        :type end_date: datetime.datetime
+        """
+        self.engagement_db_dataset = engagement_db_dataset
+        self.start_date = start_date
+        self.end_date = end_date
+
+    def to_dict(self):
+        return {
+            "engagement_db_dataset": self.engagement_db_dataset,
+            "start_date": self.start_date,
+            "end_date": self.end_date
+        }
+
```

Comment:
```suggestion
```

Reply:
I would prefer not to remove this line as it would break the formatting standard that the python community has widely adopted https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Sorry on this - I saw 3 lines here instead of 2. It's okay as is.